content/docs/command-reference/exp/save.md

```diff
@@ -73,33 +90,34 @@ status in the repo:
 ```cli
 $ git status
 On branch main
-Your branch is up to date with 'upstream/main'.
 
 Changes not staged for commit:
-  (use "git add <file>..." to update what will be committed)
-  (use "git restore <file>..." to discard changes in working directory)
 	modified:   data/.gitignore
 	modified:   src/train.py
 Untracked files:
-  (use "git add <file>..." to include in what will be committed)
 	data/new.xml.dvc
 	src/extratrees.py
-
-no changes added to commit (use "git add" and/or "git commit -a")
 ```
 
-We can run `dvc repro` to reproduce the pipeline and inspect results, e.g. by
-running `dvc metrics show`. We are not quite ready for a commit, but we want to
-save the results of this experiment, so we can run:
+We can inspect results with `dvc metrics show` (or other means) after running
+the experiment (in this case we can do so with `dvc repro` since the example
+project uses a [DVC pipeline]). We are not quite ready for a Git commit, but we
+want to save the results in the repo nonetheless:
 
 ```cli
-dvc exp save --name extra-trees -I data/new.xml.dvc -I src/extra_trees.py
+dvc exp save --name extra-trees \
+             -I data/new.xml.dvc -I src/extra_trees.py
```

Comment:
Avoids horizontal scrolling in the console block (another thing sanity checks are good for).